config/manager/manager.yaml

```diff
@@ -73,4 +73,11 @@ spec:
         args:
         - "controller"
         - --v={{.Values.controller.verbosity}}
-        - --agent-v={{.Values.agent.verbosity}}
+        - "{{if .Values.agent.hub_burst}}--agent-hub-burst={{.Values.agent.hub_burst}}{{end}}"
```

Comment:
(A) Remember that kustomize is applied to _create_ the chart template; the kustomize input has to parse as valid YAML for objects following the Kubernetes Resource Model. In other words, Go template expansion _follows_ some kube object processing, so each Go template has to be inside a string in a Kube object.

(B) I was a little surprised, but it "works" to pass empty string arguments. This does not cause any consternation. In the E2E test in CI you can see the args list for the controller in its Deployment object:

```
        - args:
          - controller
          - --v=2
          - --agent-hub-burst=7
          - ""
          - ""
          - ""
          - ""
          - ""
          - --agent-v=2
          - ""
```

You can also see the log from the controller; it has no complaints about empty strings on the command line.

You can also see the args for the agent in its Deployment in the WEC:

```
    - args:
      - agent
      - --hub-kubeconfig=/var/run/hub/kubeconfig
      - --cluster-name=cluster1
      - --addon-namespace=open-cluster-management-agent-addon
      - --hub-burst=7
      - --v=2
```